src/java/org/jivesoftware/openfire/server/LocalOutgoingServerProxy.java

```diff
@@ -0,0 +1,176 @@
+/**
+ * 
+ */
+package org.jivesoftware.openfire.server;
```

Comment:
I just want to point out that if we ever want openfire to get into debian or redhat/fedora, then we need to have a license header for every source file.